local_modules/execer/execer.cc

```diff
@@ -15,17 +15,56 @@
 #include <dirent.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <time.h>
 #include <node.h>
+#include <sstream>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string>
 #include <sys/socket.h>
+#include <sys/stat.h>
 #include <unistd.h>
 #include <v8.h>
 #include <vector>
 
 using namespace v8;
 
+bool write_response(int fd) {
+	// This is what Nginx does.
+	char buf[2048];
```

Comment:
I fixed the error, but I am going to leave the large buffer. As I said, that is what Nginx does.